app/routes/dog.js

```diff
@@ -108,5 +109,33 @@ module.exports = [
         throw e
       }
     }
+  },
+  {
+    method: 'GET',
+    path: '/dogs',
+    options: {
+      validate: {
+        query: dogsQueryParamsSchema,
+        failAction: (request, h, error) => {
+          return h.response().code(400).takeover()
+        }
+      },
+      handler: async (request, h) => {
+        let dogs = []
+
+        const sort = {
+          sortKey: request.query.sortKey,
+          sortOrder: request.query.sortOrder
+        }
+
+        if (request.query.forPurging === 'true') {
+          dogs = await getOldDogs('Exempt,Inactive,Withdrawn,Failed', sort) // , new Date(2038, 7, 2))
```

Comment:
take it we want to keep the comment?

Reply:
Yeah, for now, until I implement a bypass in later code